### ISSUE TITLE
Snackbar alert auto-dismiss timing

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#356]
Alerts should not automatically close without user interaction, or this feature should be configurable. There may be an amount of time (20 seconds maybe?) that can make auto-dismiss of snackbar alerts ok. Investigate more to see what options are.
